files/default/sql/ddl/1.1.0__initial_tables.sql

```diff
@@ -323,6 +330,22 @@ CREATE TABLE `feature_group` (
 ) ENGINE=ndbcluster AUTO_INCREMENT=13 DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `statistic_columns`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `statistic_columns` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `feature_group_id` int(11) DEFAULT NULL,
+  `name` varchar(500) COLLATE latin1_general_cs DEFAULT NULL,
```

Comment:
How did you come up with 500? 
Hive column name max length is 767 chars while for MySQL is 64. I suppose we don't enforce this limit anywhere. I the new API we should enforce it though.

Reply:
https://github.com/logicalclocks/hopsworks-chef/blob/5955801ac989d5ef29dfeb2d1e2be58cbc954d69/files/default/sql/ddl/1.1.0__initial_tables.sql#L356

I was looking at other tables where we save feature/column-names.

Reply:
Ok